buildSrc/src/main/groovy/com/uber/okbuck/rule/android/AndroidTestRule.groovy

```diff
@@ -40,6 +40,7 @@ final class AndroidTestRule extends AndroidRule {
                 aptDeps,
                 providedDeps,
                 aidlRuleNames,
+                null,
```

Comment:
Is it hard to add a simple test case? You could just try it in okbuck too. That would serve as an integration test as well so we do not break this flow. I think that should be required as part of the PR to add a simple use case for this flow.